### PR TITLE
add serapis package

### PR DESCRIPTION
**Pull request by MAfarrag**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by MAfarrag** [comment]
@conda-forge/core @conda-forge/python can you please check why the linter does not pass, it does not give any message.

**Comment by MAfarrag** [comment]
@conda-forge/python @conda-forge/core can you please check the Windows build can not install mamba for some reason.

**Comment by MAfarrag** [comment]
@conda-forge/help-python @conda-forge/python can you check, ready for review ?

**Comment by ocefpaf** [comment]
Please don't remove existing recipes. Just add yours in the PR.

File: recipes/serapis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+about:
+  home: https://github.com/MAfarrag/HAPI
+  license: BSD-3-Clause
+  license_file: LICENSE.md
+  license_family: BSD
+  license_file: LICENSE.md
+  summary: Flood modelling package
+  description: |
+    This module builds hydrodynamic model.
+  dev_url: https://github.com/Serapieum-of-alex/Serapis
```
[the remark this answers is not in the document]
Comment (MAfarrag) [reply recipes/serapis/meta.yaml]: @ocefpaf corrected

**Comment by ocefpaf** [comment]
Can you rebase or, if that is a problem for you, re-submit in a clean PR? You are still removing a file here.

**Comment by MAfarrag** [comment]
> Can you rebase or, if that is a problem for you, re-submit in a clean PR? You are still removing a file here.

well i did not delete it was added in the upstream, and i just pressed update.
So can you please merge it now before anything else is updated in the upstream ?

File: recipes/serapis/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  home: https://github.com/Serapieum-of-alex/Serapis
+  license: GPL-3.0-only
+  license_file: LICENSE.md
+  license_family: GPL-3
+  license_file: LICENSE.md
+  summary: Flood modelling package
+  description: |
+    This module builds hydrodynamic model.
+  dev_url: https://github.com/Serapieum-of-alex/Serapis
+  doc_url: https://serapis.readthedocs.io/
```
Comment (ocefpaf) [inline recipes/serapis/meta.yaml:52]: This URL is a 404 for me. Can you check if that is correct?

File: recipes/serapis/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  doc_source_url: https://github.com/Serapieum-of-alex/Serapis/main/README.md
```
Comment (ocefpaf) [inline recipes/serapis/meta.yaml:53]: Same here 404 here.

Comment (MAfarrag) [reply recipes/serapis/meta.yaml:52]: removed incorrect link

File: recipes/serapis/meta.yaml (hunk 2 above)
Comment (MAfarrag) [reply recipes/serapis/meta.yaml]: removed incorrect link

**Comment by MAfarrag** [comment]
@ocefpaf can you check now ?

File: recipes/serapis/meta.yaml [hunk 4]
```diff
@@ -49,7 +49,6 @@ about:
   description: |
     This module builds hydrodynamic model.
   dev_url: https://github.com/Serapieum-of-alex/Serapis
-  doc_url: https://serapis.readthedocs.io/
   doc_source_url: https://github.com/Serapieum-of-alex/Serapis/main/README.md
```
Comment (ocefpaf) [inline recipes/serapis/meta.yaml:52]:
```suggestion
  doc_source_url: https://github.com/Serapieum-of-alex/Serapis/blob/main/README.md
```

File: recipes/serapis/LICENSE.md [hunk 5]
```diff
@@ -0,0 +1,675 @@
+### GNU GENERAL PUBLIC LICENSE
```
Comment (ocefpaf) [inline recipes/serapis/LICENSE.md:1]: You can remove this file b/c it is in the tarball.

**Comment by MAfarrag** [comment]
@ocefpaf anything else ?

**Pull request merged**